bin/testEnvLocalK8S.sh

```diff
@@ -0,0 +1,341 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+# testEnv will setup a local test environment, for running Istio unit tests.
+
+# Based on local kubeconfig - used to reproduce the environment and to improve local testing
+
+# If GOPATH is made up of several paths, use the first one for our targets in this file
+export GO_TOP=${GO_TOP:-$(echo "${GOPATH}" | cut -d ':' -f1)}
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+export ISTIO_GO=${REPO_ROOT}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  export GOOS_LOCAL=darwin
+  echo "kube-apiserver not supported on mac, skipping."
+  exit 0
+else
+  export GOOS_LOCAL=${GOOS_LOCAL:-linux}
+fi
+
+export PATH=${GO_TOP}/bin:${PATH}
+export OUT=${GO_TOP}/out
+export ISTIO_OUT=${ISTIO_OUT:-${GO_TOP}/out/${GOOS_LOCAL}_amd64/release}
+
+# components used in the test
+export K8S_VER=${K8S_VER:-v1.9.2}
+export ETCD_VER=${ETCD_VER:-v3.2.15}
+
+export MASTER_IP=127.0.0.1
+export MASTER_CLUSTER_IP=10.99.0.1
+
+# TODO: customize the ports and generate a local config
+export KUBECONFIG=${REPO_ROOT}/tests/util/kubeconfig
+
+# Checked in certificates, to avoid regenerating them
+CERTDIR=${CERTDIR:-${ISTIO_GO}/tests/util/pki/istio-certs}
+LOG_DIR=${LOG_DIR:-${OUT}/log}
+ETCD_DATADIR=${ETCD_DATADIR:-${OUT}/etcd-data}
+
+# Ensure k8s certificates - if not found, download easy-rsa and create k8s certs
+function ensureK8SCerts() {
+    if [ -f "${CERTDIR}/apiserver.key" ] ; then
+        return
+    fi
+
+    mkdir -p "${CERTDIR}"
+    mkdir -p "${OUT}"
+    pushd "$OUT"
+    curl -L -O https://storage.googleapis.com/kubernetes-release/easy-rsa/easy-rsa.tar.gz
+    tar xzf easy-rsa.tar.gz
+    cd easy-rsa-master/easyrsa3
+
+    ./easyrsa init-pki > /dev/null
```

Comment:
this file was deleted, must have been picked up in a merge

Reply:
thanks - I'll double check what I have against master. This entire script was mostly rewritten in master, and was a little challenging to manually merge.

Reply:
just re-iterating on another pass through - this file should be deleted